modules/local/bowtie_map_mirna.nf

```diff
@@ -1,15 +1,13 @@
 // Import generic module functions
-include { saveFiles; initOptions; getSoftwareName } from './functions'
+include { saveFiles; getSoftwareName; getProcessName } from './functions'
 
 params.options = [:]
-options        = initOptions(params.options)
 
 process BOWTIE_MAP_SEQ {
-    label 'process_medium'
     tag "$meta.id"
-
+    label 'process_medium'
     publishDir "${params.outdir}",
-        mode: params.publish_dir_mode,
+        mode: params.publish_dir_mode, //TODO check for suffix, is working? needs initOptions?
```

Comment:
Is this TODO still relevant? If so do you plan on adressing it in a different PR?

Reply:
I updated the publishDir of all the bowtie local modules here: a5ae9fc